data/mods/MindOverMatter/obsolete/temporary.json

```diff
@@ -1,4 +1,14 @@
 [
+  {
+    "type": "mutation",
+    "id": "CLAIR_SPEED_READ",
```

Comment:
just to clarify, to obsolete mutation one just need to put 
```
  {
    "type": "TRAIT_MIGRATION",
    "id": "CLAIR_SPEED_READ",
    "remove": true
  }
```

Reply:
I was just going to let it fall off naturally, but perfect, this will let me fix the concentration EoC now instead of some time in the future too!